### DISCUSSION TITLE
How to disable [RemoteServer]: from appearing in the prompt when using Enter-PSSession

### DISCUSSION THREAD

**Discussion by JanDeDobbeleer** [PowerShell/PowerShell#19278] [Q&A]
A user of Oh My Posh [reports](https://github.com/JanDeDobbeleer/oh-my-posh/issues/3550#issuecomment-1454087691) they can't find out how to disable the `[RemoteServer]:` indication from the prompt when using `Enter-PSSession`. I tried to find out what exactly prints this but also wasn't very successful. Who has the answer we're looking for?

**Comment by MartinGC94** [comment]
It's consolehost that does it. See: https://github.com/PowerShell/PowerShell/blob/master/src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs#L2910 and https://github.com/PowerShell/PowerShell/blob/master/src/System.Management.Automation/engine/hostifaces/HostUtilities.cs#L646 I don't think there's any way to disable it.
